### DISCUSSION TITLE
I need help with Reusable Blocks in Gutenberg block editor

### DISCUSSION THREAD

**Discussion by sahmfz** [WordPress/gutenberg#41801] [Ideas]
Gutenberg block editor load very slowly or sometime stuck in opening depend on number of Reusable Blocks used in that page. (I used a lot 😁)

- So I wish there was an option that's allow us to prevent the rendering (preview) of the re-usable blocks in the editor. Just the name of the block in the center.

I need help in this
Thanks

**Comment by talldan** [comment]
Hi @sahmfz. What version of WordPress are you using? Do you have the Gutenberg plugin active? 

There have been some fixes attempted for this in the past.

**Reply by sahmfz** [reply]
Hi, Thank you so much for your response.
I'm using the latest version of WordPress 6.0
I don't have any Gutenberg plugin active I'm using default Gutenberg's block editor. 

I faced this issue when I tried to edit a page that has up to 10 or more Reusable Blocks in It.

![Untitled](https://user-images.githubusercontent.com/107816083/174883848-433b92f5-ea90-450b-b374-0c00be00452d.png)


there is any solution?

Thanks
